src/dns/aardvark.rs

```diff
@@ -249,9 +250,21 @@ impl Aardvark {
             .collect::<Vec<String>>()
             .join(",");
 
+        let mut dns_server: String = "".to_string();
```

Comment:
This is not required to be mutable, do it like this:
```
let y = if 12 * 15 > 150 {
    "Bigger"
} else {
    "Smaller"
};
```
From https://doc.rust-lang.org/reference/expressions/if-expr.html

Reply:
I'm not sure if I got you here, but below use-case has two nested expression so either we make `dns_server_collected` mutable or `dns_server` or otherway could to be return a result but it increases number of lines and makes it even more un-readable.

But I must be missing what you exactly have in mind so I'm cool if there any simple expression here.

Reply:
just make it another if else block, or even better just remove the isempty check since it is not needed, when it is empty it will be None anyway.

Reply:
Done.